include/benchmark/benchmark.h

```diff
@@ -543,7 +543,7 @@ class State {
   //  static void BM_Compress(benchmark::State& state) {
   //    ...
   //    double compress = input_size / output_size;
-  //    state.SetLabel(StringPrintf("compress:%.1f%%", 100.0*compression));
+  //    state.SetLabel(StringPrintF("compress:%.1f%%", 100.0*compression));
```

Comment:
this should be StrPrintF, right?

Reply:
This will be called StrFormat in absl, apparently.